(React Native) Adding a Troubleshooting section under the RN 'Upgrading' file (#71)
embrace-io/embrace-docs: docs/react-native/upgrading.md

Any place that you were previously instrumenting your app's performance using Moments can now be done using Performance
Tracing, please refer to [this guide](/react-native/features/traces/) for more information.

## Troubleshooting

> The Embrace React Native SDK implements in the shadows the Native version of our product for Android and iOS, neeading at least a sort of minimum versions required to work as expected ([more details here](/react-native/integration/))
> 
> ## Android

Reviewer: I think you could go right into the Android issue as a subheading, I moved the link to the required versions to the bottom let me know if that makes sense
```suggestion
### Android
```

> The minimum version of AGP required for the Embrace Swazzler to work as expected is `7.4.2`. If an older version is used for building the React Native Android application it will end successfully but the SDK won't be able to initialize properly, getting the following error in runtime even when everything is configured as expected:

Reviewer: ```suggestion
The minimum version of AGP required for the Embrace Swazzler to work as expected is `7.4.2`. If an older version is used for building the React Native Android application it will still build successfully but the SDK won't be able to initialize properly, getting the following error in runtime even when everything is configured as expected:
```

	at io.embrace.android.embracesdk.Embrace.start(Embrace.java:106)
	at com.ensemble3.MainApplication.onCreate(MainApplication.java:56)
	at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1192)
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6712)
	at android.app.ActivityThread.access$1300(ActivityThread.java:237)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1913)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:223)
	at android.app.ActivityThread.main(ActivityThread.java:7656)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
Caused by: java.lang.IllegalArgumentException: No appId supplied in embrace-config.json. This is required if you want to send data to Embrace, unless you configure an OTel exporter and add embrace.disableMappingFileUpload=true to gradle.properties.
	at io.embrace.android.embracesdk.internal.config.LocalConfigParser.buildConfig(LocalConfigParser.kt:112)
```

> The application will still work but the Embrace SDK won't initialize, causing unexpected issues.

Reviewer: ```suggestion
The application will still work but the Embrace SDK won't initialize, causing unexpected issues. To resolve this please ensure you have the required minimum versions set in your Android build files ([more details here](/react-native/integration/))
```
